### DISCUSSION TITLE
Error when using typescript's non-null operator in bind:this?

### DISCUSSION THREAD

**Discussion by anxpara** [sveltejs/kit#8716] [Q&A]
Is it possible to use the non-null assertion operator when binding an element? I'm getting an 'Expected }' error:

![svelte-bind-non-null-assert-operator](https://user-images.githubusercontent.com/90604943/214514474-502e6e2a-454d-4fef-8b31-1575728facff.png)

**Comment by teemingc** [comment]
Might be worth submitting this as an issue in [sveltejs/language-tools](https://github.com/sveltejs/language-tools) if it still compiles and works. I suspect it's just a VS Code extension issue.

**Reply by anxpara** [reply]
it doesn't compile 
![error](https://user-images.githubusercontent.com/90604943/214712331-db1f06dc-0d6d-4324-aac8-1544f791cd8b.png)

**Reply by teemingc** [reply]
Ah, then it's probably an issue with Svelte's parser. You'll need to submit an issue in the sveltejs/svelte repository so support for it gets implemented.

I suppose a temporary workaround is to store the result in variable? Would this work?
```svelte
<script lang="ts">
  const embeddedwf = copyColorWaypoint.stash!.embeddedwf;
</script>

<div bind:this={embeddedwf}></div>
```

**Reply by anxpara** [reply]
Yeah, I'm storing it in a temp variable in the meantime

i submitted a bug report and it got merged into this one: https://github.com/sveltejs/svelte/issues/4701
